lib/wicked_pdf/wicked_pdf_helper/assets.rb

```diff
@@ -108,6 +111,11 @@ def asset_pathname(source)
       def find_asset(path)
         if Rails.application.assets.respond_to?(:find_asset)
           Rails.application.assets.find_asset(path, :base_path => Rails.application.root.to_s)
+        elsif Rails.application.respond_to?(:assets_manifest) && Rails.application.assets_manifest.respond_to?(:find_sources)
+          asset = Rails.application.assets_manifest.find_sources(path, :base_path => Rails.application.root.to_s).first
+          if asset
+            AssetWrapper.new(asset, "#{Rails.application.assets_manifest.dir}/#{Rails.application.assets_manifest.assets[path]}")
+          end
```

Comment:
This seems pretty safe and cool the way it's implemented here, but I'm unfamiliar with how it works. Is this new in Rails 6?

Reply:
Rails 5.2 is when this came into play